packages/fluentui/docs/src/components/Playground/renderConfig.ts

```diff
@@ -30,34 +30,45 @@ type CodeSandboxImport = {
   required: boolean;
 };
 
+const newestNightlyReleaseUrlPrefix = `https://fluentsite.blob.core.windows.net/nightly-builds/newest`;
+
+const getPackageVersion = (packageName: string, latestVersion: string) => {
+  // CI build tarballs from `npm pack` and publish them nightly to blob storage for fluent ui packages
+  // When building nightly release docsite, use these urls instead of the latest package version
+  if (process.env.NIGHTLYRELEASE) {
+    return `${newestNightlyReleaseUrlPrefix}/fluentui-${packageName}-${latestVersion}.tgz`;
```

Comment:
Should this point to an exact version? For example, I will create a CodeSandbox and tomorrow it will point to a different tarball...

Reply:
```
https://fluentsite.blob.core.windows.net/nightly-builds/newest/fluentui-code-sandbox-0.52.0.tgz
https://fluentsite.blob.core.windows.net/nightly-builds/2021-02-10/fluentui-accessibility-0.52.0.tgz
```

Can you please clarify why we need `-0.52.0` suffix? It looks redundant

Reply:
Initially I thought it's good for knowing what's the latest version related to this specific nightly release. Then I saw your comment and realized we have date for this purpose. So I changed the nightly tarballs to have:
- version: 0.0.0-nightly+yyyy-mm-dd
- blob url: $(blobprefix)/yyyy-mm-dd/fluentui-$(packagename)-0.0.0-nightly.tgz

Reply:
Love it ❤️